hw/ip/flash_ctrl/data/flash_ctrl.sv.tpl

```diff
@@ -842,16 +848,23 @@ module flash_ctrl
   logic fatal_err;
   assign fatal_err = |reg2hw.fault_status;
 
+  logic fatal_std_err;
+  assign fatal_std_err = |reg2hw.std_fault_status;
+
+  lc_ctrl_pkg::lc_tx_t local_esc;
+  assign local_esc = lc_ctrl_pkg::lc_tx_bool_to_lc_tx(fatal_std_err);
 
   assign alert_srcs = { fatal_err,
+                        fatal_std_err,
                         recov_err
                       };
 
   assign alert_tests = { reg2hw.alert_test.fatal_err.q & reg2hw.alert_test.fatal_err.qe,
+                         reg2hw.alert_test.fatal_std_err.q & reg2hw.alert_test.fatal_std_err.qe,
```

Comment:
Do I understand correctly that the distinction between `fatal_err` and `fatal_std_err` will be that only the latter triggers local escalation?

Also, if `fatal_err` should be handled by SW and the alert subsystem, are we ok with leaving that as `fatal`, or should that alert channel become `recov`?

Reply:
i still want it to be fatal.  Because in an ideal world, none of that should happen.
I'm just not too sure about making them locally escalate, because that means an uncaught design bug might kill the whole system.

Reply:
Ah ok I see - this SGTM!